cornichon-kafka/cornichon-kafka.iml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
could you please remove that file :)

Reply:
removed

Attention: added *.iml to .gitignore